From dad6c62af2774d52df2d411485fe4c3929c3111d Mon Sep 17 00:00:00 2001
From: drkphnx <dark.phnx12@gmail.com>
Date: Sun, 27 Apr 2025 18:13:03 +0000
Subject: [PATCH] diable media playback progress bydefault

Signed-off-by: drkphnx <dark.phnx12@gmail.com>
---
 .../systemui/statusbar/OnGoingActionProgressController.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/OnGoingActionProgressController.java b/packages/SystemUI/src/com/android/systemui/statusbar/OnGoingActionProgressController.java
index 66e79dfdf..eb6be510e 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/OnGoingActionProgressController.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/OnGoingActionProgressController.java
@@ -471,7 +471,7 @@ public class OnGoingActionProgressController implements NotificationListener.Not
     /** Updates settings from system preferences */
     private void updateSettings() {
         mIsEnabled = Settings.System.getIntForUser(mContentResolver, ONGOING_ACTION_CHIP_ENABLED, 1, UserHandle.USER_CURRENT) == 1;
-        mShowMediaProgress = Settings.System.getIntForUser(mContentResolver, SHOW_MEDIA_PROGRESS, 1, UserHandle.USER_CURRENT) == 1;
+        mShowMediaProgress = Settings.System.getIntForUser(mContentResolver, SHOW_MEDIA_PROGRESS, 0, UserHandle.USER_CURRENT) == 1;
         
         // Read opacity as percentage (0-100)
         int opacityPercentage = Settings.System.getIntForUser(mContentResolver, PROGRESS_BAR_OPACITY, DEFAULT_OPACITY_PERCENTAGE, UserHandle.USER_CURRENT);
-- 
2.43.0

